docs/src/css/custom.css

```diff
@@ -455,17 +479,17 @@ html {
     ul,
     ol,
     blockquote {
-      @apply text-[1.2rem];
+      @apply text-[1rem];
     }
     code {
-      @apply border-0 px-3;
+      @apply border-0 px-3 align-baseline font-normal;
     }
     blockquote {
       @apply my-7;
     }
     a {
-      @apply underline;
-      /* color: var(--ifm-color-primary ); */
+      @apply no-underline hover:underline;
```

Comment:
Since the color choice is poor, it will be impossible to figure out which part of the text is a link.
That's why it was underlined in the first place.